content/docs/1.6.0/concepts.md

```diff
@@ -64,14 +64,14 @@ The Engine and replicas are orchestrated using Kubernetes.
 In the figure below,
 
 - There are three instances with Longhorn volumes.
-- Each volume has a dedicated controller, known as the Longhorn Engine. For a V1 volume, the engine runs as a Linux process, while for a V2 volume, it operates as an SPDK logical volume.
+- Each volume has a dedicated controller, called the Longhorn Engine. For V1 volumes, the engine runs as a Linux process, while for V2 volumes, it operates as an SPDK RAID block device (bdev).
```

Comment:
Do we need to explain what a v2 replica is?

Reply:
Yes, added. Thank you.